spice/tests/vm_listen.py

```diff
@@ -47,7 +47,7 @@ def run(vt_test, test_params, env):
         test_params['start_vm'] = "yes"
         if test_params['spice_port_closed'] == "yes":
             cmd = "nc -l %s" % test_params['spice_port']
-            nc_process = process.SubProcess(cmd)
+            nc_process = process.get_sub_process_klass(cmd)(cmd)
```

Comment:
We will re-write this.
calling function on returning object. (one)(one)

Let it stay open, until we have time to review.